src/verso/Verso/Doc/Elab/Monad.lean

```diff
@@ -416,6 +423,22 @@ unsafe def inlineExpandersForUnsafe (x : Name) : DocElabM (Array InlineExpander)
 @[implemented_by inlineExpandersForUnsafe]
 opaque inlineExpandersFor (x : Name) : DocElabM (Array InlineExpander)
 
+def substForDocReconstructionMvars (same : Expr) (e : Expr) : MetaM Expr := do match e with
```

Comment:
Why is the parameter named `same`? What's the intuition there?

Reply:
Once upon a time I was using MetaM to force the mvars to be equal, so all the mvars were getting identified with the same thing, hence "same." But this is unenlightening; `replacement` is better than `same`.